packages/drivers/odsp-driver/src/test/epochTests.spec.ts

```diff
@@ -17,7 +17,7 @@ import { getHashedDocumentId } from "../odspPublicUtils";
 import { IVersionedValueWithEpoch, persistedCacheValueVersion } from "../contracts";
 import { mockFetchOk, mockFetchSingle, createResponse } from "./mockFetch";
 
-const createUtLocalCache = () => new LocalPersistentCache(2000);
+const createUtLocalCache = () => new LocalPersistentCache();
```

Comment:
I wonder if it would be better or worse to just create the cache once and reuse it in all the tests.  It would let us actually exercise the capability to manage different documents' entries independently.

Eh, except the same docId is used every time, so slightly less interesting?  Just thinking out loud, probably shouldn't change it.

Reply:
yes, not sure if it is worthy it.